backend/services/API.js

```diff
@@ -8,13 +8,21 @@ const reqTimeout = 1000 * 5;
  * specific amount of data according to searchTerm.
  */
 async function getSearchedImages (apiType, limit, searchTerm, offset) {
-  const { link, key, limitString, offsetString } =
-    apiType === apiSources.giphy.id ? apiSources.giphy : apiSources.pixabay;
 
-  const searchQuery = `&q=${searchTerm}`;
-  const query = `&${limitString}=${limit}&${offsetString}=${offset}`;
-  const url = `${link}${key}${searchQuery}${query}`;
+  let url;
```

Comment:
my code editor complains about spacing here. Could you please check if your linting is working as projects setup?